guides/common/assembly_importing-content.adoc

```diff
@@ -9,13 +9,79 @@ include::modules/proc_creating-a-custom-product.adoc[leveloffset=+1]
 include::modules/proc_adding-custom-rpm-repositories.adoc[leveloffset=+1]
 
 ifndef::satellite[]
-include::modules/proc_adding-custom-rpm-repositories-for-alma-linux-9.adoc[leveloffset=+1]
+:os_major: 9
+:gpg_url: http://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-{os_major}
+:os_name: Alma Linux
+:os_name_anchor: Alma_Linux
```

Comment:
I wonder if you can do replacements on a variable in asciidoc. Every time the anchor is `os_name` with ` ` and `.` replaced by `_`. Not a blocker, but I wonder if you thought about this.

Reply:
I will look into this, but not for this PR. A quick search did not reveal anything usefull.